src/services/main.service.ts

```diff
@@ -142,6 +143,14 @@ export class MainService {
     }
   }
 
+  async loadBlockStyles(block: BlockMapping) {
+    try {
+      await this.loadCSS(`${window.hlx.codeBasePath}/dist/${block.name}/${block.name}.css`);
```

Comment:
wenn man das integriert geht man davon aus das jeder block eine eigene *.css datei hat oder ? was passiert wenn ein block keine *.css datei hat ?

Reply:
wird den catch error in der console geworfen und die app bricht erstmal nicht zusammen... ich weiß es nicht jetzt wie ich einen check machen kann ob die style datei definiert ist... ich muss mir was überlegen...